### ISSUE TITLE
provide instructions for wiring up a remote component

### ISSUE THREAD

**Issue by batwicket** [matthiasn/systems-toolbox#21]
It would be nice if there were instructions wrt wiring a remote http-kit (and/or immutant or other ring-compatible server, perhaps) server into the toolbox for websocket exchanges. Perhaps a parameter in the instantiation of :server/ws-cmp in example/core.clj?

**Comment by matthiasn** [comment]
Yeah, I was thinking about spending the entire next week on documentation and a book update describing the mechanism in detail. How would an instantiation parameter help here in your opinion? Cheers

**Comment by batwicket** [comment]
Directly using a server address/port in the example's wiring code might save some effort for toolbox developers. Ideally the code will be so intuitive that they won't have to read any comments/documentation!

**Comment by wrabbit-revisited** [comment]
btw.. i'm sure you're ahead of me on this, but the legacy tendency is to divide frameworks like Rails into client/server. Perhaps the toolbox, taking advantage of new capabilities like websockets, is a set of tools for peers.. whether they be browser, console, desktop, server, or mobile. The directory organization for the example may still be a bit confusing to someone that doesn't understand that cljs and is for peers running JS, clj is for peers running Java (well.. the JVM for now) and cljc is a way to share similar concepts/functionality across all peers.
